public/assets/sass/pages/resources/important-links.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../util/_sass-mixins";
+@import "../../util/variables";
+
+section {
+    max-width: 600px;
+    margin: auto;
+    text-align: center;
+
+    .list {
+        list-style-type: none;
+        padding-left: 0;
+
+        .list-item-content {
+            border: 2px solid var(--mathsoc-pink-2);
+            border-radius: 3em;
+            margin: 1em;
+
+            .list-item-link {
+                text-decoration: none;
+                display: flex;
+                align-items: center;
+                justify-content: space-between;
+                padding: 1em;
+
+                .list-item-icon {
+                    height: 100%;
+                    width: auto;
+                    padding-right: 1em;
+                }
+                
+                .list-item-title {
+                    margin: 0;
+                    font-weight: 400;
+                    color: var(--mathsoc-pink-2);
+                    padding-left: 1em;
+                }
+            }
+        }
+
+        .list-item-content:hover {
```

Comment:
We'll need to ensure that keyboard-only users are accounted for here. Not all users navigate by mouse!

Ideally, if you try using `tab` to navigate the page (as people with physical disabilities or repetitive strain injuries might), you should be able to do so while having as similar to an experience to mouse-using users as possible. If you try to do so on the current version of the page, you'll be able to see that the buttons don't get highlighted when you focus on them 😔 

It's an easy enough fix though - try adding the styles to `.list-item-content:focus-visible` as well as `.list-item-content:hover`, and most of the issue should be resolved (you might have to tweak a few minor things to get both mouse and keyboard navigation to have the same effect). 

(see the above comment for an example :) )